src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/umbblockgridentries.component.js

```diff
@@ -589,13 +515,6 @@
                     
                     const oldIndex = evt.oldIndex;
                     vm.movingLayoutEntry = contextVM.getLayoutEntryByIndex(oldIndex);
-                    if(vm.movingLayoutEntry.forceLeft ||  vm.movingLayoutEntry.forceRight) {
-                        // if one of these where true before, then we made a change here:
-                        vm.blockEditorApi.internal.setDirty();
-                    }
-                    vm.movingLayoutEntry.forceLeft = false;
-                    vm.movingLayoutEntry.forceRight = false;
-                    vm.movingLayoutEntry.$block.__scope.$evalAsync();// needed for the block to be updated
```

Comment:
Did you mean to remove the last line here? It seems sort of related to force left/right, but is it only related to that?

Reply:
Yep, forceLeft forceRight has an impact on the DOM, so that was the reason to call $evalAsync